### PR TITLE
Adding Wio 3G and BG96

### PR DESCRIPTION
**Pull request by ytsuboi**
CC @toyowata

### CODE REVIEW

File: test/info.py [hunk 1]
```diff
@@ -58,6 +58,8 @@
     ("lpc11u35_archble_if",                         False,      0x0000,     "bin"       ),
     ("lpc11u35_archpro_if",                         False,      0x0000,     "bin"       ),
     ("lpc11u35_archmax_if",                         False,      0x0000,     "bin"       ),
+    ("lpc11u35_wio3g_if",                           False,      0x0000,     "bin"       ),
+    ("lpc11u35_wiobg96_if",                         False,      0x0000,     "bin"       ),
```
[the remark this answers is not in the document]
Comment (brianesquilona) [reply test/info.py]: Conflict in current board id [info.py](https://github.com/ARMmbed/DAPLink/blob/master/test/info.py#L210)
`(   0x9014,     'lpc11u35_96b_nitrogen_if',                 None,               None                                    ), # TODO - set target to 'Seeed-96Boards-Nitrogen' when mbed-os supports this`

[the remark this answers is not in the document]
Comment (toyowata) [reply test/info.py]: @brianesquilona 
`WIO_3G` (9014) and `WIO_BG96` (9015) are already registered in our platform database (assigned by TAM) and the Mbed OS tools below:

https://github.com/ARMmbed/mbed-os-tools/blob/master/src/mbed_os_tools/detect/platform_database.py#L311-L312

[the remark this answers is not in the document]
Comment (brianesquilona) [reply test/info.py]: > @brianesquilona
> `WIO_3G` (9014) and `WIO_BG96` (9015) are already registered in our platform database (assigned by TAM) and the Mbed OS tools below:
> 
> https://github.com/ARMmbed/mbed-os-tools/blob/master/src/mbed_os_tools/detect/platform_database.py#L311-L312

@toyowata you can add this board ID in SUPPORTED_CONFIGURATIONS and change the  lpc11u35_96b_nitrogen_if to 0916 to remove the conflict. @maclobdell @ashok-rao, any comments on this?

**Comment by toyowata** [comment]
@brianesquilona thanks for the ID change in the info.py

File: test/info.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashok-rao) [reply test/info.py]: LGTM!

**Pull request merged**